vispy/app/canvas.py

```diff
@@ -512,6 +514,16 @@ def __repr__(self):
                 % (self.__class__.__name__,
                    self.app.backend_name, hex(id(self))))
 
+    def _ipython_display_(self):
```

Comment:
As mentioned in the meeting, I think this is the old interface for this. There is likely a `_repr_*_` equivalent for newer jupyter versions. Maybe https://ipython.readthedocs.io/en/stable/config/integrating.html#rich-display

Reply:
IPywidget only implements this in bleeding edge - the latest release still uses `_ipython_display_`. See [this commit](https://github.com/jupyter-widgets/ipywidgets/commit/7ae97acc96ce9f78bd43dbd0778cd7db36804966#) that adds it (merged after latest release).

Reply:
Let's proxy both.

Reply:
The pull request was merged January of 2020: https://github.com/jupyter-widgets/ipywidgets/pull/2021
The latest release is January of 2021: https://pypi.org/project/ipywidgets/

Reply:
That's ... odd. What you say is true, but the 7.6.3 release contains the version from *before* that commit. I double-checked by opening the release tarball.

Reply:
Ah well ... it should work regardless of the version now.

Reply:
They have a `7.x` branch that I think they cherry-pick fixes/merges from the master branch. The master branch is their dev/0.8 branch currently I think. That's my best guess.